ADR/0022-secret-mgmt-for-user-workloads.md

```diff
@@ -95,12 +175,11 @@ spec:
 
 #### Additions to Roles and Permissions
 
-|     Role      | Permissions             | API Groups                | Verbs                                   | Resources
-|---------------|-------------------------|---------------------------|-----------------------------------------|----------------------------------------------------------------------
-| Contributor   | RemoteSecret            | appstudio.redhat.com      | get, list, watch                        | remotesecret
-| Maintainer    | RemoteSecret            | appstudio.redhat.com      | get, list, watch, create, update, patch | remotesecret
-| Admin         | RemoteSecret            | appstudio.redhat.com      | get, list, watch, create, update, patch | remotesecret
-
+| Role        | Permissions  | API Groups           | Verbs                                   | Resources    |
+|-------------|--------------|----------------------|-----------------------------------------|--------------|
+| Contributor | RemoteSecret | appstudio.redhat.com | get, list, watch                        | remotesecret |
+| Maintainer  | RemoteSecret | appstudio.redhat.com | get, list, watch, create, update, patch | remotesecret |
```

Comment:
Since maintainers are not permitted to create regular secrets today, the create will have no effect. Should we delay the introduction of these permissions until an alternate way to introduce secret data is available  or they are able to create secrets?

Reply:
It may be confusing indeed.
It seems that only Admins can normally deal with the Secrets now, right?

If so, I'd vote to decrease Maintainer permissions (to Contributor level) for the time.